### PR TITLE
Kg links row wise

### PR DESCRIPTION
**Pull request by rijulvohra**
Small updates to `get-kg-links` to output top k candidates row-wise.

### CODE REVIEW

File: tl/cli/get-kg-links.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def add_arguments(parser):
                         default=5,
                         help='desired number of output candidates per input cell.')
 
-    parser.add_argument('--k-rows', action='store_true', type=bool, dest='k_rows', required=False,
+    parser.add_argument('--k-rows', action='store', type=bool, dest='k_rows', required=False,
```
Comment (saggu) [inline tl/cli/get-kg-links.py:32]: why did you change from store_true ? @rijulvohra

**Review by saggu — CHANGES_REQUESTED** [review]
Please update this option in the readme as well

File: tl/cli/get-kg-links.py (hunk 1 above)
Comment (rijulvohra) [reply tl/cli/get-kg-links.py:32]: @saggu It was giving the follwing error on running the command `tl get-kg-links -h`
Screen Shot 2021-04-13 at 12 05 23 PM

Comment (saggu) [reply tl/cli/get-kg-links.py:32]: ah, so remove the type parameter from the parameter definition
